bower.json

```diff
@@ -10,7 +10,7 @@
     "tests"
   ],
   "dependencies": {
-    "n-ui-foundations": "^2.5.0",
+    "n-ui-foundations": "^3.0.0",
```

Comment:
There is a resolution below which will need to be removed 🔍

Reply:
Oh thank you! I will do that.